src/sentry/static/sentry/app/views/userFeedback/projectUserFeedback.jsx

```diff
@@ -50,6 +52,13 @@ const ProjectUserFeedback = createReactClass({
   },
 
   componentWillMount() {
+    // Redirect any Sentry 10 user that has followed an old link and ended up here
+    const {organization, location, params: {orgId}} = this.props;
+    const hasSentry10 = new Set(organization.features).has('sentry10');
+    if (hasSentry10) {
+      browserHistory.replace(`/organizations/${orgId}/user-feedback/${location.search}`);
```

Comment:
Do we want to set the project?

Reply:
Keeping the projectid in the filter would be better.